From e4667f11df3b755c6fdd2eb71f8bbd7db2c74e8b Mon Sep 17 00:00:00 2001
From: Alejandro Sierra <b18039@freescale.com>
Date: Thu, 9 Jun 2016 16:16:03 -0500
Subject: [PATCH 11/11] Add qspi2 and m4fastup defconfigs for mx6sxscm

Add qspi2 and m4fastup defconfigs for:
    - mx6sxscm_1gb_evb
    - mx6sxscm_512mb_evb
    - mx6sxscm_epop_evb

Signed-off-by: Alejandro Sierra <b18039@freescale.com>
---
 configs/mx6sxscm_1gb_evb_m4fastup_defconfig   | 7 +++++++
 configs/mx6sxscm_1gb_evb_qspi2_defconfig      | 7 +++++++
 configs/mx6sxscm_512mb_evb_m4fastup_defconfig | 7 +++++++
 configs/mx6sxscm_512mb_evb_qspi2_defconfig    | 7 +++++++
 configs/mx6sxscm_epop_evb_m4fastup_defconfig  | 7 +++++++
 configs/mx6sxscm_epop_evb_qspi2_defconfig     | 7 +++++++
 6 files changed, 42 insertions(+)
 create mode 100644 configs/mx6sxscm_1gb_evb_m4fastup_defconfig
 create mode 100644 configs/mx6sxscm_1gb_evb_qspi2_defconfig
 create mode 100644 configs/mx6sxscm_512mb_evb_m4fastup_defconfig
 create mode 100644 configs/mx6sxscm_512mb_evb_qspi2_defconfig
 create mode 100644 configs/mx6sxscm_epop_evb_m4fastup_defconfig
 create mode 100644 configs/mx6sxscm_epop_evb_qspi2_defconfig

diff --git a/configs/mx6sxscm_1gb_evb_m4fastup_defconfig b/configs/mx6sxscm_1gb_evb_m4fastup_defconfig
new file mode 100644
index 0000000..ca9fcdc
--- /dev/null
+++ b/configs/mx6sxscm_1gb_evb_m4fastup_defconfig
@@ -0,0 +1,7 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6sxscm/imximage_lpddr2.cfg,MX6SX,SYS_BOOT_QSPI,SYS_AUXCORE_FASTUP"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6SXSCM=y
+CONFIG_SYS_MALLOC_F=y
+CONFIG_SYS_MALLOC_F_LEN=0x400
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6sxscm_1gb_evb_qspi2_defconfig b/configs/mx6sxscm_1gb_evb_qspi2_defconfig
new file mode 100644
index 0000000..a31ff2e
--- /dev/null
+++ b/configs/mx6sxscm_1gb_evb_qspi2_defconfig
@@ -0,0 +1,7 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6sxscm/imximage_lpddr2.cfg,MX6SX,SYS_BOOT_QSPI"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6SXSCM=y
+CONFIG_SYS_MALLOC_F=y
+CONFIG_SYS_MALLOC_F_LEN=0x400
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6sxscm_512mb_evb_m4fastup_defconfig b/configs/mx6sxscm_512mb_evb_m4fastup_defconfig
new file mode 100644
index 0000000..a2f30c0
--- /dev/null
+++ b/configs/mx6sxscm_512mb_evb_m4fastup_defconfig
@@ -0,0 +1,7 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6sxscm/imximage_lpddr2.cfg,MX6SX,512MB_LPDDR2,SYS_BOOT_QSPI,SYS_AUXCORE_FASTUP"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6SXSCM=y
+CONFIG_SYS_MALLOC_F=y
+CONFIG_SYS_MALLOC_F_LEN=0x400
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6sxscm_512mb_evb_qspi2_defconfig b/configs/mx6sxscm_512mb_evb_qspi2_defconfig
new file mode 100644
index 0000000..14c7427
--- /dev/null
+++ b/configs/mx6sxscm_512mb_evb_qspi2_defconfig
@@ -0,0 +1,7 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6sxscm/imximage_lpddr2.cfg,MX6SX,512MB_LPDDR2,SYS_BOOT_QSPI"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6SXSCM=y
+CONFIG_SYS_MALLOC_F=y
+CONFIG_SYS_MALLOC_F_LEN=0x400
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6sxscm_epop_evb_m4fastup_defconfig b/configs/mx6sxscm_epop_evb_m4fastup_defconfig
new file mode 100644
index 0000000..7eb53c3
--- /dev/null
+++ b/configs/mx6sxscm_epop_evb_m4fastup_defconfig
@@ -0,0 +1,7 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6sxscm/imximage_lpddr2.cfg,MX6SX,512MB_LPDDR2,MX6SXSABRESD_EMMC_REWORK,SYS_BOOT_QSPI,SYS_AUXCORE_FASTUP"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6SXSCM=y
+CONFIG_SYS_MALLOC_F=y
+CONFIG_SYS_MALLOC_F_LEN=0x400
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
diff --git a/configs/mx6sxscm_epop_evb_qspi2_defconfig b/configs/mx6sxscm_epop_evb_qspi2_defconfig
new file mode 100644
index 0000000..be30620
--- /dev/null
+++ b/configs/mx6sxscm_epop_evb_qspi2_defconfig
@@ -0,0 +1,7 @@
+CONFIG_SYS_EXTRA_OPTIONS="IMX_CONFIG=board/freescale/mx6sxscm/imximage_lpddr2.cfg,MX6SX,512MB_LPDDR2,MX6SXSABRESD_EMMC_REWORK,SYS_BOOT_QSPI"
+CONFIG_ARM=y
+CONFIG_TARGET_MX6SXSCM=y
+CONFIG_SYS_MALLOC_F=y
+CONFIG_SYS_MALLOC_F_LEN=0x400
+CONFIG_DM=y
+CONFIG_DM_THERMAL=y
-- 
1.9.1

